pkg/apis/pingcap/v1alpha1/validation/validation_test.go

```diff
@@ -230,9 +230,85 @@ func TestValidateRequestsStorage(t *testing.T) {
 	}
 }
 
+func TestValidateService(t *testing.T) {
+	g := NewGomegaWithT(t)
+	tests := []struct {
+		name                     string
+		loadBalancerSourceRanges []string
+		expectedErrors           int
+	}{
+		{
+			name:                     "correct LoadBalancerSourceRanges",
+			loadBalancerSourceRanges: strings.Split("192.168.0.1/32", ","),
+			expectedErrors:           0,
+		},
+		{
+			name:                     "incorrect LoadBalancerSourceRanges",
+			loadBalancerSourceRanges: strings.Split("192.168.0.1", ","),
+			expectedErrors:           1,
```

Comment:
I suggest you check if the error message is as expected，thanks

Reply:
Because I'm not sure the fieldpath is correct

Reply:
update